core/client/lemverse.js

```diff
@@ -455,6 +455,11 @@ Template.lemverse.onDestroyed(function () {
 Template.lemverse.helpers({
   allRemoteStreamsByUsers: () => peer.remoteStreamsByUsers.get(),
   guest: () => Meteor.user({ fields: { 'profile.guest': 1 } })?.profile.guest,
+  onboarded: () => {
+    const onboarded = Meteor.users.findOne(Meteor.userId)?.status?.onboarded;
+
+    return onboarded !== undefined ? onboarded : true;
```

Comment:
Can't you just do something like this?

```suggestion
    return Meteor.user({ fields: {'status.onboarded': 1 } })?.status.onboarded || false;
```

Reply:
No, we should check if it's `undefined` because it's different than `false`. If the value onboarded doesn't exist, it's 
`undefined` this mean it's an older created account, but if the value is `onboarded: false`, it's a new registration and should have the onboarding steps

Reply:
But thank's for the tip, I used it to retrieve the onboarded value :)